### PR TITLE
Container Day post

### PR DESCRIPTION
**Pull request by jnoller**
Woooo @everett-toews pls review @kmsholcomb (time sensitive)

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-e3c849c3ad/)

**Comment by kmsholcomb** [comment]
I'll look at it right now, @jnoller.

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "OpenStack Summit Austin: Containers Meetup!"
+date: 2016-04-13
+comments: true
+author: Jesse Noller <jesse.noller@rackspace.com>
+published: true
+excerpt: >
+  You're invited to join the Carina team Tuesday, April 24th for a full
```
Comment (everett-toews) [inline _posts/2016-04-13-container-day-austin-summit.md]: April 26

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  afternoon of hands on hacking, talks in a meetup style atmosphere alongside
+  the OpenStack Austin summit.
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: ... Carina team on Tuesday, April 26, for ...       (note the date!)
... afternoon of hands-on hacking and talks in a meetup-style atmosphere ...

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  - Rackspace
+  - OpenStack
+  - News
+authorIsRacker: true
+---
+
+![ContainUrself]({% asset_path 2016-04-13-container-day-austin-summit/ContainYourself.png %})
+
+{{ page.excerpt }} We've got some great news - with the OpenStack summit joining
+us in Austin the week of April 25th, we've worked with a lot of amazing people to
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: ...some great news: with ...
... the week of April 25, ...

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+get a dedicated demo/discussion/hack space for containers!
+
+For this year’s OpenStack Summit, [Rackspace wanted to create a place](http://blog.rackspace.com/relax-recharge-rackspace-cantina-openstack-summit-austin/)
+where attendees could power down and chill, or power up. Any OpenStack Summit
+attendee can come join us across the street from the summit to geek out about
+containers, Docker, Docker Swarm, etc for all three days (April 25th-27th).
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: ... Docker Swarm, etc. for all three days (April 25-27).

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+I'm glad you asked that. Below is a fully interactive 'Google Map' that reveals
+the hidden rebel base.
+
+<iframe style="border: 0;" src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3446.0847637412344!2d-97.74326968537538!3d30.263165981803215!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8644b5a8186ac2f1%3A0xc2e20ce9aa35bd25!2s333+E+2nd+St%2C+Austin%2C+TX+78701!5e0!3m2!1sen!2sus!4v1460473290564" width="600" height="450" frameborder="0" allowfullscreen="allowfullscreen"></iframe>
+
+### Join us!
+
+Grab a t-shirt, have a drink and a bite to eat, and chat containers with us. Hack
+on applications learn, experiment. It's all fair! We're geeks, so we're not
+focused on ABC (always be closing).
```
Comment (everett-toews) [inline _posts/2016-04-13-container-day-austin-summit.md]: always be containerizing

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+{{ page.excerpt }} We've got some great news - with the OpenStack summit joining
+us in Austin the week of April 25th, we've worked with a lot of amazing people to
+get a dedicated demo/discussion/hack space for containers!
+
+For this year’s OpenStack Summit, [Rackspace wanted to create a place](http://blog.rackspace.com/relax-recharge-rackspace-cantina-openstack-summit-austin/)
+where attendees could power down and chill, or power up. Any OpenStack Summit
+attendee can come join us across the street from the summit to geek out about
+containers, Docker, Docker Swarm, etc for all three days (April 25th-27th).
+
+The Carina team has a dedicated space with screens, tables and the team to focus
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: ... screens, tables, and the team ...

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-6c9cbf572f/)

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+on building, creating, and discussing all things containers.
+
+### But it gets cooler!
+
+What's great is that we're not just inviting Summit attendees! We're inviting
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: We're not just inviting Summit attendees! We're also inviting *any other container/Carina interested people* to join us Tuesday, April 26, from 12 pm-5 pm for a dedicated meetup afternoon.

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+Openstack Summit attendees *and any other container / Carina interested people*
+to join us *Tuesday April 26th from 12pm-5pm* for a dedicated meet-up afternoon.
+
+We have a full array of speakers, including:
+
+* [Ross Kukulinski](https://twitter.com/rosskukulinski) of Yodlr talk about *Using Carina in Production*
+* [Lee Calcote](https://twitter.com/lcalcote) talking about *Characterizing and Comparing Container Orchestrators.*
+* [Tim Hockin](https://twitter.com/thockin) senior software engineer @ Google talk about *Pain Management for Container Apps*.
+* Our own [Everett Toews](https://twitter.com/everett_toews) talking about *Effective Docker Swarm*.
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: - [Ross Kukulinski](https://twitter.com/rosskukulinski), of Yodlr, talking ...
- [Tim Hockin](https://twitter.com/thockin), senior software engineer at Google, talking ...

(Delete the periods from the ends of the last three bullets)

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+
+### Ermahgersh! how do I sign up if I'm not a Summit attendee?
+
+If you're a Summit attendee, you're in like Flynn. If not, just
+[sign up here, for free](http://www.cvent.com/d/pfq4wl/4W) so we know we have
+enough food, drink, stickers etc. Y'all are welcome, summit attendee or no!
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: If you're a Summit attendee, you're already in. If not, just [sign up here, for free](http://www.cvent.com/d/pfq4wl/4W), so we can have enough food, drink, stickers, etc. All are welcome ...

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+
+[No seriously sign up here](http://www.cvent.com/d/pfq4wl/4W).
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: [Seriously, sign up here](http://www.cvent.com/d/pfq4wl/4W).

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+
+### Where do I go?
+
+I'm glad you asked that. Below is a fully interactive 'Google Map' that reveals
+the hidden rebel base.
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: I'm glad you asked that. Here is a fully interactive Google Map that reveals the location of the hidden rebel base.

File: _posts/2016-04-13-container-day-austin-summit.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+
+<iframe style="border: 0;" src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3446.0847637412344!2d-97.74326968537538!3d30.263165981803215!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8644b5a8186ac2f1%3A0xc2e20ce9aa35bd25!2s333+E+2nd+St%2C+Austin%2C+TX+78701!5e0!3m2!1sen!2sus!4v1460473290564" width="600" height="450" frameborder="0" allowfullscreen="allowfullscreen"></iframe>
+
+### Join us!
+
+Grab a t-shirt, have a drink and a bite to eat, and chat containers with us. Hack
+on applications learn, experiment. It's all fair! We're geeks, so we're not
```
Comment (kmsholcomb) [inline _posts/2016-04-13-container-day-austin-summit.md]: Hack on applications, learn, and experiment.

**Comment by kmsholcomb** [comment]
Done, @jnoller. Mostly just commas. :)

**Comment by jnoller** [comment]
thanks @kmsholcomb @everett-toews changed

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-698fa1f514/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-8b1496cb5c/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-3c5e2a7259/).

**Pull request merged**